libs/trades/src/lib/trades-data-provider.ts

```diff
@@ -53,8 +53,8 @@ export const sortTrades = (trades: TradeFields[]) => {
   );
 };
 
-const update = (data: TradeFields[], delta: TradeFields[]) =>
-  produce(data, (draft) => {
+const update = (data: TradeFields[], delta: TradeFields[]) => {
```

Comment:
chore (non-blocking): another of these patterns, feel free to mark it resolved, just taking a note in case it turns out to be relevant

Reply:
This was requested by Matt, he prefer `"arrow-body-style": ["error", "always"]`